js/logging.js

```diff
@@ -91,10 +91,10 @@ loggingModule.factory(
     "applicationLoggingService",
     ["$log","$window", "LOGGING_CONFIG", function($log, $window, LOGGING_CONFIG){
         return({
-            error: function(message){
+            error: function(message, desc){
                 if (LOGGING_CONFIG.LOGGING_TYPE !== 'none') {
                   // preserve default behaviour
-                  $log.error.apply($log, arguments);
```

Comment:
Beforehand the call was basically doing `console.error([message])`. When I then appended `desc`, the output was  `console.error([message, desc])`, which looked weird. I believe we only want to show the message on the console.